saos-enrichment/src/main/java/pl/edu/icm/saos/enrichment/apply/JudgmentEnrichmentService.java

```diff
@@ -48,11 +51,26 @@
             return null;
         }
         
+        entityManager.detach(judgment);
```

Comment:
Please, add a comment above the method that the returned judgment will be in a detached state and explain why it is necessary?